lib/slashfeed.json

```diff
@@ -14,6 +14,12 @@
     "fields": [
         {
             "name": "Block",
+            "description": "Shows the current block information.",
+            "main": "/feed/latest-block",
```

Comment:
@rbndg We discussed whether or not to keep an aggregated field `Block` to save network roundtrips for users who want the entire block at once, let me know if I should remove it, at least from `slashfeed.json`.